config/gasSchedule.go

```diff
@@ -244,6 +244,7 @@ func FillGasMap_EthereumAPICosts(value uint64) map[string]uint64 {
 
 func FillGasMap_BigIntAPICosts(value uint64) map[string]uint64 {
 	gasMap := make(map[string]uint64)
+	gasMap["EllipticCurveNew"] = value
```

Comment:
where do you have all the API cost for all the EC functions ? Here you should have more than 10 new elements